themes/default/gulpfile.js

```diff
@@ -221,6 +236,8 @@ gulp.task('make-js', ['make-js-components', 'make-js-npm'], function() {
 		.pipe(gulp.dest('js'))
 });
 
+gulp.task('build', ['pure', 'make-css', 'cms-css', 'make-js', 'svgo']);
```

Comment:
Should svgo not also be added to watch? And this task should have json in it. Probably we could then change watch to be: 

    gulp.task('watch', ['build', 'browserSync'], function () {